pygfx/utils/cube_camera.py

```diff
@@ -0,0 +1,95 @@
+from ..linalg import Vector3
+from ..objects._base import WorldObject
+from ..cameras import PerspectiveCamera
+from ..resources import Texture
+from ..renderers import WgpuRenderer
+
+
+def _is_cube_texture(texture):
+    if not isinstance(texture, Texture):
+        return False
+    return texture.dim == 2 and len(texture.size) == 3 and texture.size[2] == 6
+
+
+class _CubeCameraRenderer(WgpuRenderer):
+    def __init__(self, target, blend_mode="default"):
+
+        assert _is_cube_texture(target), "target must be a cube texture"
+
+        super().__init__(target, blend_mode=blend_mode)
+
+        # Pre generate views of different layers of the cube texture
+        self._target_views = []
+        for layer in range(6):
+            self._target_views.append(
+                target.get_view(view_dim="2d", layer_range=range(layer, layer + 1))
+            )
+
+    def render(self, scene: WorldObject, camera, layer):
+        # all _target_views are "TextureView" from the same Texyure and have same size, so we can change "_target" here safely.
+        self._target = self._target_views[layer]
+        super().render(scene, camera)
+
+
+class CubeCamera(WorldObject):
+    """
+    Create a camera array to help create a cube texture map from a viewpoint in a scene.
+
+    Note that the texture data will be written directly to the internal "GPUTexture" object,
+    not to the "data" attribute of target. That is, its data cannot be accessed from the CPU.
+    """
+
+    def __init__(self, target, near=0.1, far=1000, blend_mode="default"):
+        super().__init__()
+
+        self.renderer = _CubeCameraRenderer(target, blend_mode=blend_mode)
```

Comment:
One small comment: could you please make this `self._renderer` and create a `@property` for `.renderer`so that this attribute is also listed in the docs?